.github/workflows/test.yaml

```diff
@@ -50,4 +52,6 @@ jobs:
         run: make test-coverage
 
       - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3
+        uses: codecov/codecov-action@v4
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
I believe this will failed for forked repos, as secrets are not accessible to forks

Reply:
Nevermind, I saw this log in the action:

```
The PR is happening in a forked repo. Using tokenless upload.
```

Reply:
They allow PRs to be tokenless https://github.com/codecov/codecov-action?tab=readme-ov-file#breaking-changes